srcpkgs/budgie-desktop/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'budgie-desktop'
```

Comment:
This will need to be patched in order to build with mutter 3.36 unless upstream tags a release by the time this PR is merge ready.
Please also add [these](https://github.com/not-chicken/void-packages/commit/d8db510737bdb1781d0a541c30e97414f4e4bc44) minor changes to the template.

Reply:
cherry-picked. Thanks !